install/aiven/01_create_monitoring_user.mdx

```diff
@@ -5,10 +5,26 @@ backlink_href: /docs/install
 backlink_title: 'Installation Guide'
 ---
 
-import Step01CreateMonitoringUserAiven from "./_01_create_monitoring_user.mdx"
+import MonitoringUserSetupInstructions from "../../components/MonitoringUserSetupInstructions";
```

Comment:
These wrapper components are no longer necessary with this new approach. Instead, inline the respective `_...` components into the main ones that included them.